libsrc/api/JsonAPI.cpp

```diff
@@ -393,11 +393,53 @@ void JsonAPI::handleCommand(const JsonApiCommand& cmd, const QJsonObject &messag
 		sendErrorReply("The command is deprecated, please use the Hyperion Web Interface to configure", cmd);
 	break;
 		// END
+	case Command::GetCurrentImage:
+		handleGetCurrentImageCommand(message, cmd);
+	break;
 	default:
 	break;
 	}
 }
 
+void JsonAPI::handleGetCurrentImageCommand(const QJsonObject &message, const JsonApiCommand& cmd)
+{
+	QString replyMsg;
+	
+	Debug(_log, "Get image command received");
+	QString savePath = message["path"].toString();
+	
+	Debug(_log,"Save path: %s", QSTRING_CSTR(savePath));
+	
+	int priority = _hyperion->getCurrentPriority();
+	const PriorityMuxer::InputInfo priorityInfo = _hyperion->getPriorityInfo(priority);
+	Image<ColorRgb> image = priorityInfo.image;
+
+	QImage jpgImage(reinterpret_cast<const uchar*>(image.memptr()), image.width(), image.height(), qsizetype(3) * image.width(), QImage::Format_RGB888);
+	QByteArray byteArray;
+	QBuffer buffer(&byteArray);
+	buffer.open(QIODevice::WriteOnly);
+	jpgImage.save(&buffer, "JPG");
```

Comment:
There should be an error reply, if the Image in target format failed to be created